scripts/entities/actors/actor.gd

```diff
@@ -0,0 +1,21 @@
+extends Entity
+class_name Actor
+
+@export var turn_state: WorldTurnBase.StateMachine.STATES = WorldTurnBase.StateMachine.STATES.player
```

Comment:
<img width="438" alt="Screenshot 2024-12-11 at 9 04 13 PM" src="https://github.com/user-attachments/assets/0adad4a8-1736-4fb4-a6a0-6a88e505dab9" />
(i feel bad riding yall on this but this is one example where it's kinda important. Apologies 🩷